data/base/base-route.php

```diff
@@ -0,0 +1,258 @@
+<?php
+namespace Elementor\Data\Base;
+
+use WP_REST_Server;
+
+abstract class BaseRoute {
```

Comment:
```suggestion
abstract class Base_Route {
```

Reply:
So the same is with sub-controller? All_Children, etc?
i don't understand when to use '_', eg what is the difference
elementor/core/document-types/page-base.php

Reply:
Handled: https://github.com/elementor/elementor/pull/12820/commits/8f488da8e79062fddfc8efbab4b94872b5d4f3e0